device-types/MikroTik/CCR1016-12S-1S+.yaml

```diff
@@ -0,0 +1,40 @@
+manufacturer: MikroTik
+model: CCR1016-12S-1S+
+slug: ccr1016-12s-1splus
+interfaces:
+  - name: sfp1
+    type: 1000base-x-sfp
+  - name: sfp2
+    type: 1000base-x-sfp
+  - name: sfp3
+    type: 1000base-x-sfp
+  - name: sfp4
+    type: 1000base-x-sfp
+  - name: sfp5
+    type: 1000base-x-sfp
+  - name: sfp6
+    type: 1000base-x-sfp
+  - name: sfp7
+    type: 1000base-x-sfp
+  - name: sfp8
+    type: 1000base-x-sfp
+  - name: sfp9
+    type: 1000base-x-sfp
+  - name: sfp10
+    type: 1000base-x-sfp
+  - name: sfp11
+    type: 1000base-x-sfp
+  - name: sfp12
+    type: 1000base-x-sfp
+  - name: sfp-sfpplus1
```

Comment:
Is this the actual interface name? I'm not familiar with Mikrotik so just checking.

Reply:
Thanks for your fast reply Jeremy. MikroTik names their interfaces like that (yep, I know it's weird). This is an example output from a router I have here
```
[rootmin@ccr1072] > inte eth pr
Flags: X - disabled, R - running, S - slave
 #    NAME                             MTU MAC-ADDRESS       ARP             SWITCH
 0    ether1                          1500 6C:3B:6B:E2:3E:5F enabled
 1 R  sfp-sfpplus1                    1500 6C:3B:6B:E2:3E:57 enabled
 2    sfp-sfpplus2                    1500 6C:3B:6B:E2:3E:58 enabled
 3 R  sfp-sfpplus3                    1500 6C:3B:6B:E2:3E:59 enabled
 4    sfp-sfpplus4                    1500 6C:3B:6B:E2:3E:5A enabled
 5    sfp-sfpplus5                    1500 6C:3B:6B:E2:3E:5B enabled
 6    sfp-sfpplus6                    1500 6C:3B:6B:E2:3E:5C enabled
 7    sfp-sfpplus7                    1500 6C:3B:6B:E2:3E:5D enabled
 8    sfp-sfpplus8                    1500 6C:3B:6B:E2:3E:5E enabled
```